CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### New Features and Improvements:
 
+- Add '--clone-cluster-enable-audit-logs' argument which enables audit logging on cloned clusters.
```

Comment:
Nit: should we make it clear that audit logging is by default off on the cloned cluster, regardless if the original cluster has it on? Or is that the usual assumption anyways?

Reply:
I added some elaboration to this statement.